azure_pipeline.yaml

```diff
@@ -114,6 +114,17 @@ stages:
           agentPool: ${{ variables.agentPool }}
           terraformVersion: ${{ variables.terraformVersion }}
 
+      - template: pipeline-templates/terraform-plan.yaml
```

Comment:
I think we should probably rename the yaml files,

![image](https://user-images.githubusercontent.com/21194782/76606918-028c0d80-650b-11ea-8b5b-9651f0c78dcc.png)

all.yaml made sense when it was the only configuration

how about backend-load-balancing-config.yaml?

unless you can think of something shorter?

Reply:
done